### DISCUSSION TITLE
Trigger smcb authentication flow in RU returns "Not Found"

### DISCUSSION THREAD

**Discussion by fue-medatixx** [Digitaler-Impfnachweis/certification-apis#28] [Q&A] [answered]
Trigger smcb authentication flow in RU returns "Not Found".
Request: https://id.ru.impfnachweis.info/auth/realms/ubirch-certificator/protocol/openid-connect/auth?redirect_uri=connector://authenticated&response_type=code&scope=openid&client_id=cert-auth&nonce=123456789

What could be the problem?

**Comment by fue-medatixx** [comment]
Even after the client_id was changed to "user-access-ti" in the flow description TI SMCB, we get NOT FOUND back in the RU: Error: Realm does not exist.

**Comment by mge-medisoftware** [comment, marked as answer]
I confirm this:  400 bad request "Realm does not exist"

**Reply by fue-medatixx** [reply]
The Spec has a new realm:
https://id.ru.impfnachweis.info/auth/realms/ubirch-certify/protocol/openid-connect/auth

It works now for us!

**Reply by oliver-steinbrecher** [reply]
Sorry for this failure - we've fixed the setup and it should be more stable now.
